src/telemetry/index.ts

```diff
@@ -0,0 +1,69 @@
+import {
+    TelemetryRecorderProvider as BaseTelemetryRecorderProvider,
+    MarketingTrackingTelemetryProcessor,
+    NoOpTelemetryExporter,
+    type MarketingTrackingProvider,
+    type TelemetryEventMarketingTrackingInput,
+} from '@sourcegraph/telemetry'
+
+import { GraphQLTelemetryExporter } from './gqlTelemetryExporter'
+import { getSessionTracker } from './sessionTracker'
+import { getUserTracker } from './userTracker'
+
+function getTelemetrySourceClient(): string {
+    return 'about.web'
+}
+
+/**
+ * TelemetryRecorderProvider is the default provider implementation for the
+ * Sourcegraph web app.
+ */
+export class TelemetryRecorderProvider extends BaseTelemetryRecorderProvider<'', ''> {
+    constructor() {
+        super(
+            {
+                client: getTelemetrySourceClient(),
+            },
+            new GraphQLTelemetryExporter('https://sourcegraph.com'),
+            [new MarketingTrackingTelemetryProcessor(new TrackingMetadataProvider())],
+            {
+                /**
+                 * Use buffer time of 100ms - some existing buffering uses
+                 * 1000ms, but we use a more conservative value.
+                 */
+                bufferTimeMs: 100,
+                bufferMaxSize: 10,
```

Comment:
I think we should disable buffering for the about site, I'm not sure the mechanism is very well-proven yet (IIRC someone from one of the client teams had taking a closer look at buffering on their backlog)

Reply:
```suggestion
                /**
                 * Disable buffering for now
                 */
                bufferTimeMs: 0,
```